docs/getting-started-guides/ubuntu.md

```diff
@@ -118,7 +118,7 @@ The `FLANNEL_NET` variable defines the IP range used for flannel overlay network
 
 After all the above variables being set correctly, we can use following command in cluster/ directory to bring up the whole cluster.
 
-`$ KUBERNETES_PROVIDER=ubuntu ./kube-up.sh`
+`$ KUBERNETES_PROVIDER=ubuntu ../kube-up.sh`
```

Comment:
Ic.
Yet i do agree that text description could be missed easily, as in your case.
U R free to make it more readable anyway.